src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListPermissions.java

```diff
@@ -0,0 +1,53 @@
+package ca.corefacility.bioinformatics.irida.ria.web.linelist;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.sample.Sample;
+import ca.corefacility.bioinformatics.irida.security.permissions.project.ProjectOwnerPermission;
+import ca.corefacility.bioinformatics.irida.security.permissions.sample.UpdateSamplePermission;
+
+/**
+ * Component to standardize user permissions for samples on the
+ * linelist page.
+ */
+@Component
+public class LineListPermissions {
```

Comment:
I have been trying more and more to moving items like this to separate `Components`, since the `autowired` dependencies are specific for this component.  I do not have a problem with renaming this to `PermissionComponent` and moving it to a more appropriate spot that other controllers can use.